src/test/java/com/github/sdonkov/PrefixTreeBenchmark.java

```diff
@@ -0,0 +1,57 @@
+package com.github.sdonkov;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.infra.Blackhole;
+import org.openjdk.jol.info.GraphLayout;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
```

Comment:
Is this decided by your IDE? I guess there's a threshold of imported classes over which it decides to import everything.
It'd be best to turn it off and import everything explicitly. Wildcard imports are fine if they are for test stuff like the annotations here or assert methods in unit tests, but hurt readability otherwise.